DATA/production/qc-async/fv0.json

```
@@ -21,26 +21,48 @@
         "conditionDB": {
           "url": "qcdb.cern.ch:8083"
         }
-      },
-      "tasks": {
-        "FV0Digits": {
-          "active": "true",
-          "taskName": "Digits",
-          "className": "o2::quality_control_modules::fv0::DigitQcTask",
-          "moduleName": "QcFV0",
-          "detectorName": "FV0",
-          "cycleDurationSeconds": "60",
-          "maxNumberCycles": "-1",
-          "dataSource": {
-            "type": "direct",
-            "query": "digits:FV0/DIGITSBC/0;channels:FV0/DIGITSCH/0"
-            },
-          "taskParameters": {
-            "ChannelIDs": "0,1,2,3,4,5,6,7,8,9,10,11"
-          }
+    },
+    "tasks": {
+      "DigitQcTask": {
+        "active": "true",
+        "className": "o2::quality_control_modules::fv0::DigitQcTask",
+        "moduleName": "QcFV0",
+        "detectorName": "FV0",
+        "cycleDurationSeconds": "600",
+        "maxNumberCycles": "-1",
+		"saveObjectsToFile":"QC_FV0.root",
+        "dataSource": {
+          "type": "direct",
+          "query": "digits:FV0/DIGITSBC/0;channels:FV0/DIGITSCH/0"
+        },
+        "taskParameters": {
+          "trgModeInnerOuterThresholdVar": "Ampl",
+		   "trgThresholdNChannels": "2",
+		   "trgThresholdCharge": "8",
+		   "trgThresholdChargeInner": "4",
+		   "trgThresholdChargeOuter": "4"
         }
       }
     },
-    "dataSamplingPolicies": [
-    ]
+    "checks": {
+      "CFDEffCheck": {
+        "active": "true",
```

```suggestion
      "FV0CFDEff": {
        "active": "true",
        "checkName": "CFDEff",
```
Also here the new naming convention could be applied